x-pack/plugins/siem/public/components/timeline/helpers.tsx

```diff
@@ -11,14 +11,19 @@ import { convertKueryToElasticSearchQuery, escapeQueryValue } from '../../lib/ke
 
 import { DataProvider, DataProvidersAnd, EXISTS_OPERATOR } from './data_providers/data_provider';
 
+const converTimestampFieldToQuery = (value: string | number) =>
+  `@timestamp: ${isNumber(value) ? value : new Date(value).valueOf()}`;
+
 const buildQueryMatch = (dataProvider: DataProvider | DataProvidersAnd) =>
   `${dataProvider.excluded ? 'NOT ' : ''}${
     dataProvider.queryMatch.operator !== EXISTS_OPERATOR
-      ? `${dataProvider.queryMatch.field} : ${
-          isNumber(dataProvider.queryMatch.value)
-            ? dataProvider.queryMatch.value
-            : escapeQueryValue(dataProvider.queryMatch.value)
-        }`
+      ? dataProvider.queryMatch.field === '@timestamp'
```

Comment:
beyond `@timestamp`, should this logic apply to all fields of type `date`?

Reply:
I do not know all the field but wait I can ask the index pattern and if it is a date I can do that